modules/ROOT/pages/questions/question-09.adoc

```diff
@@ -0,0 +1,14 @@
+9. It’s noted in https://ephocks.atlassian.net/browse/TINY-8312[TINY-8312]
+   that:
+
+   > text patterns are on by default `text_patterns: false` or
+   > `text_patterns: []` will turn the functionality off
+
+   Is this a change from previously?
+
+   Or, if not, has the means of turning the functionality off (if
+   desired) changed from previously?
+
+   And, if it has changed, does this change constitute something an
+   upgrader needs to be aware of? That is, does this change need to be
+   noted in the _Migration Guide_?
```

Comment:
Apologies, this doubles up with my reply above, but:

Before, if you wanted Text Pattern functionality you had to opt in by including it as a plugin. Now, it's part of core, so the functionality is included and "on" by default. In order to opt out, you now need to use the `text_patterns` option, setting it to either `false` or `[]`.  

So yes, will need to be in the migration guide.